climada/hazard/test/test_base.py

```diff
@@ -1104,6 +1105,35 @@ def test_vector_to_raster_pass(self):
         self.assertTrue(haz_fl.fraction.min() >= 0)
         self.assertTrue(haz_fl.fraction.max() <= 0.5 / 2)
 
+
+class TestClear(unittest.TestCase):
+    """Test clear method"""
+
+    def test_clear(self):
+        """Clear method clears everything"""
+        haz1 = Hazard('TC')
+        haz1.read_excel(HAZ_TEMPLATE_XLS)
```

Comment:
Manually set some more attributes (such as the `units` attribute) to make sure that they are actually touched by `clear`. 
```suggestion
        haz1.read_excel(HAZ_TEMPLATE_XLS)
        haz1.units = "m"
        haz1.foo = np.arange(10)
```

Reply:
On that one: haz1.foo will be reset to an empty numpy array. Are we fine with that or do we want all non-standard attributes of the hazard object to be removed entirely from the object?

Reply:
Since the `clear` function from the base class is used in all derived classes, it's currently the easiest way to support additional attributes, because we don't currently define which attributes are considered "standard". So, for this PR, I think we should stick with this behavior. In the future, we might be defining something like "standard" attributes for derived Hazard classes etc.